portable/IAR/AtmelSAM7S64/lib_AT91SAM7X128.h

```diff
@@ -1,1469 +1,1396 @@
+    {
+        /* Disable output */
+        pPio->PIO_ODR = inputEnable;
+        pPio->PIO_PER = inputEnable;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_CfgOpendrain */
+/** \brief Configure PIO in open drain */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_CfgOpendrain( AT91PS_PIO pPio,              /* \arg pointer to a PIO controller */
+                                          unsigned int multiDrvEnable ) /* \arg pio to be configured in open drain */
+    {
+        /* Configure the multi-drive option */
+        pPio->PIO_MDDR = ~multiDrvEnable;
+        pPio->PIO_MDER = multiDrvEnable;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_CfgPullup */
+/** \brief Enable pullup on PIO */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_CfgPullup( AT91PS_PIO pPio,            /* \arg pointer to a PIO controller */
+                                       unsigned int pullupEnable ) /* \arg enable pullup on PIO */
+    {
+        /* Connect or not Pullup */
+        pPio->PIO_PPUDR = ~pullupEnable;
+        pPio->PIO_PPUER = pullupEnable;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_CfgDirectDrive */
+/** \brief Enable direct drive on PIO */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_CfgDirectDrive( AT91PS_PIO pPio,           /* \arg pointer to a PIO controller */
+                                            unsigned int directDrive ) /* \arg PIO to be configured with direct drive */
+
+    {
+        /* Configure the Direct Drive */
+        pPio->PIO_OWDR = ~directDrive;
+        pPio->PIO_OWER = directDrive;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_CfgInputFilter */
+/** \brief Enable input filter on input PIO */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_PIO_CfgInputFilter( AT91PS_PIO pPio,           /* \arg pointer to a PIO controller */
+                                            unsigned int inputFilter ) /* \arg PIO to be configured with input filter */
+
+    {
+        /* Configure the Direct Drive */
+        pPio->PIO_IFDR = ~inputFilter;
+        pPio->PIO_IFER = inputFilter;
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_PIO_GetInput */
+/** \brief Return PIO input value */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_PIO_GetInput( /* \return PIO input */
+        AT91PS_PIO pPio )                     /* \arg  pointer to a PIO controller */
```

Comment:
Should this be one line?

Reply:
Probably? It's a bit weird how this looks, but it's how they have it now. All this is doing is swapping the C++ style comment out to C style ones.